griptape/cli/core/cloud_client.py

```diff
@@ -162,6 +167,38 @@ def create_app(self, app_data: dict, environment_id: Optional[str]):
         url = urljoin(self.endpoint_url, f"environments/{environment_id}/apps")
         return self._send_request("POST", url, app_data)
 
-    def create_deployment(self, deployment_data: dict, app_id: str):
+    def create_deployment(self, deployment_data: dict, app_id: str) -> Response:
         url = urljoin(self.endpoint_url, f"apps/{app_id}/deployments")
         return self._send_request("POST", url, deployment_data)
+
+    def create_run(self, run_data: dict, app_id: str) -> Response:
+        url = urljoin(self.endpoint_url, f"apps/{app_id}/runs")
+        return self._send_request("POST", url, run_data)
+
+    def get_run(self, run_id: str) -> Response:
+        url = urljoin(self.endpoint_url, f"runs/{run_id}")
+        return self._send_request("GET", url)
+
+    def poll_run(self, run_id: str) -> None:
+        """
+        Poll a Griptape App Run on the Cloud.
+        """
+
+        run = self.get_run(run_id).json()
+        event_ids = set()
+        while run["status"] not in ["SUCCEEDED", "FAILED"]:
```

Comment:
does it make sense to have a timeout here or be able to pass a timeout value? im fine with saying that is an enhancement for later

Reply:
I think it's ok not to have a timeout because Runs themselves have a timeout, and users can always stop the CLI command.